src/configMap.ts

```diff
@@ -26,10 +26,37 @@ export function loadConfigMapData(obj: KubernetesFileObject) {
     const uri_str = `${uriScheme}://${obj.resource}/${encoded_data}/${obj.id}`;
     const uri = vscode.Uri.parse(uri_str);
     vscode.workspace.openTextDocument(uri).then(
-    (doc) => {
-        vscode.window.showTextDocument(doc);
-    },
-    (error) => {
-        vscode.window.showErrorMessage('Error loading data file: ' + error);
+        (doc) => {
+            vscode.window.showTextDocument(doc);
+        },
+        (error) => {
+            vscode.window.showErrorMessage('Error loading data file: ' + error);
+        });
+}
+
+export async function deleteKubernetesConfigFile(kubectl: Kubectl, obj: KubernetesFileObject, explorer: KubernetesExplorer) {
+    const confirm = 'Delete';
+    let result = await vscode.window.showWarningMessage(`Are you sure you want to delete ${obj.id} this can not be undone`, confirm, 'Cancel');
```

Comment:
We have elsewhere used `extension.ts > deleteMessageItems` to give the Cancel button the close affordance though I'm not sure whether this makes any difference in the new VS Code notification UI.

Reply:
done.